internal/policies/mount/adsys_mount/src/test.rs

```diff
@@ -0,0 +1,30 @@
+#[cfg(test)]
+mod tests {
+    use crate::{parse_entries, MountEntry};
+
+    /// Tests parse_entries function.
+    #[test]
+    fn test_parse_entries() {
```

Comment:
I think there are multiple tests that we can do, like empty files, and so on… (also, file that doesn’t exist). That way, you can test the Err case which is not covered right now.

While I’m at it… Do you mind looking if we can transform those in some kind of table testing? (Honest request, I haven’t looked how to do that in rest, but at worst, we can have a for loop basically, without real subtest support, and relax the assert to a global error?)